src/Microsoft.DotNet.Interactive.Tests/KernelSchedulerTests.cs

```diff
@@ -311,7 +311,7 @@ public void Infinite_loops_can_be_cancelled()
             .ThrowAsync<OperationCanceledException>();
     }
 
-    [Fact]
+    [Fact(Skip = "requires System.Runtime.ControlledExecution")]
     public async Task After_an_infinite_loop_is_cancelled_the_scheduler_can_still_be_used()
```

Comment:
@colombod This was one of the tests that was deadlocking in my experiment to add testing against `net472`. Based on what we were looking at earlier, I am guessing the explanation may be that the `ControlledExecution` API is not available in `net472`?